ethereum/referencetests/build.gradle

```diff
@@ -32,14 +32,17 @@ dependencies {
   implementation project(':services:kvstore')
 
   implementation 'com.fasterxml.jackson.core:jackson-databind'
+  implementation 'com.google.code.findbugs:jsr305'
+  implementation 'com.google.guava:guava'
+
 
   testOutput sourceSets.test.output
 }
 
 task ('validateReferenceTestSubmodule') {
   description = "Checks that the reference tests submodule is not accidentially changed"
   doLast {
-    def expectedHash = 'b8abaebf5949877ead03e8f74ac03c0ede41d5f7'
```

Comment:
There is no need to update the reference tests as part of tuweni.  Please remove changes to this file and the submodule update form the PR.

Reply:
That must be me making a mess again. I'll fix that.